src/types/HttpServer.ts

```diff
@@ -0,0 +1,9 @@
+import { Request } from 'express';
+
+export type RootHandlerRequest = Request & {
+    body?: {
+        id: string;
+        method: string;
+        params: Array<unknown>;
+    };
+};
```

Comment:
the only thing is that I'd find clearer to just see  a file like HttpServer.types.ts instead of repeating the same name inside a nested directory. But I'm happy to discuss.

Reply:
I tried to follow the same structure that is in the projects itself, but we need to decide an strategy for this since I think that there is no a standardised way to do it.

Reply:
updated